nix/machines/flake-module.nix

```diff
@@ -40,7 +40,6 @@ in
         ./_common/users.nix
         ./devServer/default.nix
         ./devServer/hardware-configuration.nix
-        inputs.microvm.nixosModules.host
```

Comment:
Bonus cleanup here?

Reply:
exactly, Im going to have some additional PRs related to #https://github.com/socallinuxexpo/scale-network/issues/771 thatll be doing more of the same